src/teleop/gui/src/components/ControlGUI.vue

```diff
@@ -2,27 +2,49 @@
 <div>
     <DriveControls></DriveControls>
     <ArmControls></ArmControls>
-    <GimbalControls></GimbalControls>>
-</div>
+    <GimbalControls></GimbalControls>
+    <JointStateTable v-bind:JointStateData="JointState" v-bind:vertical="true"></JointStateTable>
+    <MoteusStateTable v-bind:MoteusStateData="MoteusState"></MoteusStateTable>
 
+</div>
 </template>
 
 <script>
-    import DriveControls from './DriveControls.vue';
-    import ArmControls from './ArmControls.vue';
-    import GimbalControls from './GimbalControls.vue';
+import ROSLIB from "roslib"
+import DriveControls from './DriveControls.vue';
+import ArmControls from './ArmControls.vue';
+import GimbalControls from './GimbalControls.vue';
+import JointStateTable from './JointStateTable.vue';
+import MoteusStateTable from './MoteusStateTable.vue'
+
+export default {
+    data() {
+        return {
+            JointState: {},
```

Comment:
can we make these be camel-case? i.e jointState and moteusState? and generally use camel case throughout the code?